tests/test_integration.py

```diff
@@ -19,3 +25,29 @@ def test_compile_contract_with_different_name_than_file(project):
     file_name = "DifferentNameThanFile.sol"
     contract = project.contracts["ApeDifferentNameThanFile"]
     assert contract.source_id == file_name
+
+
+def test_get_imports(project, compiler):
+    import_dict = compiler.get_imports(TEST_CONTRACT_PATHS, BASE_PATH)
+    contract_imports = import_dict["Imports.sol"]
+    # NOTE: make sure there aren't duplicates
+    assert len([x for x in contract_imports if contract_imports.count(x) > 1]) == 0
+    # NOTE: returning a list
+    assert type(contract_imports) == list
+    # NOTE: in case order changes
+    assert EXPECTED_IMPORTS_SET == set(contract_imports)
+
+
+def test_get_import_remapping(compiler, project, config):
+    import_remapping = compiler.get_import_remapping()
+    assert import_remapping
+
+    with config.using_project(project.path / "project"):
```

Comment:
When I change with `config.using_project(project.path / "project") as sub_project`
and then on the line below, do:

```python
dependencies = sub_project.dependencies
assert dependencies
```

it seems to fix one issue for me at least